map/generate-offline-map-with-local-basemap/README.md

```diff
@@ -1,8 +1,8 @@
 # Generate offline map with local basemap
 
-Take a web map offline, but instead of downloading an online basemap, use one which is already on the device.
+Use the `OfflineMapTask` to take a web map offline, but instead of downloading an online basemap, use one which is already on the device.
```

Comment:
We should avoid using class names in this section, would suggest something like the following:
```suggestion
Take a web map offline using a local basemap on the device.
```